lib/taskana-core/src/test/java/acceptance/workbasket/WorkbasketQueryWithOrderedPaginationAccTest.java

```diff
@@ -28,7 +28,7 @@ public WorkbasketQueryWithOrderedPaginationAccTest() {
         super();
     }
 
-    @Ignore
+    @Disabled
```

Comment:
This test does not work?

Reply:
Yes. Created new ticket to look at disabled tests.